pymoto/modules/io.py

```diff
@@ -352,3 +356,69 @@ def _response(self, *args):
             f.write("\n")
 
         self.iter += 1
+
+
+class TransientToVTI(Module):
```

Comment:
Maybe name `SeriesToVTI` more general? Although I don't really know what other series than 'time' there would be...